app/views/hosts/console/vnc.html.erb

```diff
@@ -9,11 +9,22 @@
           {:id => "troubleshootingButton", :class => "btn btn-default"}
         )
  ) %>
-<p class="toggle-hidden-value"><%= _('Password:') %> <a href="#" class="toggle-hidden-value"><%=icon_text("plus")%></a><span><%= @console[:password] %></span></p>
-<div id='vnc' data-port='<%= @console[:proxy_port] %>' data-password='<%= @console[:password] %>' data-encrypt='<%= @encrypt %>'>
-  <div id="noVNC_status_bar" class="console-status">
-    <div id="noVNC_status" class="col-md-12 label">Loading ...</div>
-  </div>
+<% if @console[:proxy_port] %>
```

Comment:
Would it makes sense to do this on the controller side so you only have a `port`? In the future I would actually hope we introduce an API to get console access and it would be nice if it could use the same structures as the controllers sends.